tst/DepartmentManagerTest.java

```diff
@@ -0,0 +1,4 @@
+package tst;
```

Comment:
This will represent an interesting case in the future when we write a command/script to test your code. I'm making a note for our sessions